### DISCUSSION TITLE
feat: support for custom httpx.client

### DISCUSSION THREAD

**Discussion by brandonkzw** [langfuse/langfuse#1071] [Support]
Current version of Langfuse Python SDK (2.7.6) does not support custom httpx.client, i.e. for developers that are dealing with self-signed certificate, this may cause an issue for https connection.

Will be creating a PR for this feat.

**Comment by brandonkzw** [comment]
Forked and implemented on commit https://github.com/brandonkzw/langfuse-python/commit/053a07ffc60e5cda6d999ff7902feb348a17ea52

**Comment by maxdeichmann** [comment]
@brandonkzw thanks a lot for the contribution! I just merged this :)

**Comment by ishan00** [comment]
Hello, which version of langfuse (pip package) resolves this? Getting the SSL error for `2.60.4`
